cmsplugin_filer_image/templates/cmsplugin_filer_image/image.html

```diff
@@ -13,8 +13,24 @@
               });
             })(jQuery)
         </script>
-      {% endaddtoblock %}
-    {% endif %}
+      {% endif %}
+
+      {% if overlay_link %}
+        <script type="text/javascript">
+            (function($){
+                $('#{{rnd}}').click(function(){
+                    var imgURL = $(this).data('rel');
+                    $('body').append(
+                        '<div class="frame" style="z-index: 10002; position: fixed; left: 0; top: 0; height: 100%; width: 100%; text-align: center; background-color: rgba(0, 0, 0, 0.88);">'+
```

Comment:
Any reason why these styles are inline and not in an external css file or a style tag?